src/workerd/jsg/modules.h

```diff
@@ -52,6 +52,108 @@ class CommonJsModuleContext: public jsg::Object {
   jsg::Value exports;
 };
 
+class NodeJsModuleContext: public jsg::Object {
```

Comment:
Naming — Node.js **compat** module

Reply:
This name is not exposed directly to end users so the `compat` part here is optional.